<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QListWidget</class>
 <widget class="QMainWindow" name="QListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QListWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>157</width>
         <height>444</height>
        </rect>
       </property>
       <attribute name="label">
        <string>QListWidget</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="btnList">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>list</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/images/128.bmp</normaloff>:/icons/images/128.bmp</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnPlus">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>plus</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/images/316.bmp</normaloff>:/icons/images/316.bmp</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnInsert">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>insert</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/images/318.bmp</normaloff>:/icons/images/318.bmp</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDelete">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/images/delete1.bmp</normaloff>:/icons/images/delete1.bmp</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnSelect">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>select</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/images/124.bmp</normaloff>:/icons/images/124.bmp</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <attribute name="label">
        <string>QTreeWidget</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>157</width>
         <height>444</height>
        </rect>
       </property>
       <attribute name="label">
        <string>QTableWidget</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>QListWidget</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>item:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QToolButton" name="btnUnall">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>unselect all</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="btnAll">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>select all</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QToolButton" name="btnInverse">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>inverse</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="5">
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>hello world</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>good</string>
            </property>
            <property name="checkState">
             <enum>PartiallyChecked</enum>
            </property>
           </item>
           <item>
            <property name="text">
             <string>think</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
           </item>
           <item>
            <property name="text">
             <string>idea</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>editable</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>QTreeWidget</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>QTableWidget</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlist"/>
   <addaction name="actionplus"/>
   <addaction name="actioninsert"/>
   <addaction name="actiondelete"/>
   <addaction name="actionselect"/>
   <addaction name="actionclear"/>
   <addaction name="separator"/>
   <addaction name="actionexit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionlist">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/128.bmp</normaloff>:/icons/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>list</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/delete1.bmp</normaloff>:/icons/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionplus">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/316.bmp</normaloff>:/icons/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>plus</string>
   </property>
  </action>
  <action name="actioninsert">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/316.bmp</normaloff>:/icons/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>insert</string>
   </property>
  </action>
  <action name="actionselect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/124.bmp</normaloff>:/icons/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/exit.bmp</normaloff>:/icons/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionPopMenu">
   <property name="text">
    <string>&lt;---select---&gt;</string>
   </property>
  </action>
  <action name="actionselect_all">
   <property name="text">
    <string>select_all</string>
   </property>
  </action>
  <action name="actionunselect_all">
   <property name="text">
    <string>unselect_all</string>
   </property>
  </action>
  <action name="actioninverse">
   <property name="text">
    <string>inverse</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/105.bmp</normaloff>:/icons/images/105.bmp</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
